tests/data/composition_no_trailing_comma.py

```diff
@@ -165,20 +165,17 @@ def tricky_asserts(self) -> None:
             _C.__init__.__code__.co_firstlineno + 1,
         )
 
-        assert (
-            expectedexpectedexpectedexpectedexpectedexpectedexpectedexpectedexpect
-            == {
-                key1: value1,
-                key2: value2,
-                key3: value3,
-                key4: value4,
-                key5: value5,
-                key6: value6,
-                key7: value7,
-                key8: value8,
-                key9: value9
-            }
-        )
+        assert expectedexpectedexpectedexpectedexpectedexpectedexpectedexpectedexpect == {
```

Comment:
This line exceeds the line length limit, so there is a problem to resolve here.

Reply:
Going by the current [`black` code style](https://github.com/psf/black/blob/b8c1020b526b488a1a216d9a4d58fc8616b61a99/docs/the_black_code_style.md#line-length) documentation, I think it's OK for the line length to be exceeded, under some circumstances.  Whether this qualifies as an acceptable situation for that exception, I don't know and would be glad for feedback on.